components/HorizontalFeed/HorizontalFeed.tsx

```
@@ -61,14 +66,20 @@ const HorizontalFeed: FC<HorizontalFeedProps> = ({ feeds }) => {
+          {Array.from({ length: feeds.length + 1 }).map((_, i) => (
+            <>
+              {i < feeds.length ? (
+                <Feed
+                  key={i}
+                  feed={feeds[i]}
+                  hovered={isHovered(i)}
+                  step={calculateStep(i, feeds)}
+                  height={getHeight(i)}
+                />
+              ) : (
+                <FetchMoreInspector fetchMore={fetchMore} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)